app/models/models.py

```diff
@@ -46,6 +46,24 @@ def get_users(cursor):
 
         return results
 
+    @staticmethod
+    def login_user(cursor, email, password):
+        """
+        method to check if user can login
+        """
+        query = "SELECT * FROM users WHERE (email=%s);"
+        cursor.execute(query, [email])
+        user = cursor.fetchone()
```

Comment:
repeating this multiple times in your code
might be worth wrapping it in its own function and make it alittle more dynamic so as to use it in a number of places
i.e
```python
def fetch_one(query, item):
	cursor.execute(query, [item])
	element = cursor.fetchone()
	return element
```